languages/ocaml.scm

```diff
@@ -394,6 +416,7 @@
     (field_get_expression)
     (labeled_argument)
     (module_path)
+    (module_type_path)
```

Comment:
Shouldn't `extended_module_path` also be here, then?

Reply:
It was just above in the list. I made the equivalence classes more explicit in the new version